src/currencyservice/Dockerfile

```diff
@@ -14,22 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine
+FROM alpine as builder
 
-RUN apk update
-RUN apk add git cmake make g++ grpc-dev re2-dev protobuf-dev c-ares-dev
+RUN apk update && apk add git cmake make g++ grpc-dev protobuf-dev
 
 ARG OPENTELEMETRY_CPP_VERSION=1.10.0
```

Comment:
@styblope as you are touching this file, could you update the CPP to latest version?
Today (29/11) it is `1.12.0`.